src/components/SpeakerHome.tsx

```diff
@@ -1,5 +1,163 @@
+          <motion.div
+            className="flex gap-4 md:gap-4 lg:gap-6"
+            key={currentIndex}
+            initial={{ x: direction > 0 ? 300 : -300, opacity: 0 }}
+            animate={{ x: 0, opacity: 1 }}
+            exit={{ x: direction > 0 ? -300 : 300, opacity: 0 }}
+            transition={{ duration: 0.5, ease: 'easeInOut' }}
+          >
+            {visibleSpeakers.map((speaker, i) => (
+              <div
+                className="min-w-[260px] min-h-[300px] md:min-w-[180px] md:min-h-[220px] lg:min-w-[220px] lg:min-h-[260px] flex-shrink-0"
+                key={speaker.id}
+              >
+                <SpeakerCard
+                  key={i}
+                  name={speaker.name}
+                  title={speaker.title}
+                  imageUrl={speaker.imageUrl}
+                  isActive={activeIndex === i} 
+                  handleHover={() => setActiveIndex(i)}
+                  currentIndex={currentIndex}
+                  pos={i}
+                />
+              </div>
+            ))}
+          </motion.div>
+        </div>
+
+        {/* Mobile: 1 card */}
+        <div className="flex md:hidden relative mt-8 sm:mt-10 justify-center items-center h-[320px] sm:h-[360px] overflow-hidden">
+          <AnimatePresence mode="wait">
+            <motion.div
+              key={currentIndex} // ensures a re-render on speaker change
+              initial={{ opacity: 0, x: 100 }}
+              animate={{ opacity: 1, x: 0 }}
+              exit={{ opacity: 0, x: -100 }}
+              transition={{ duration: 0.5, ease: 'easeInOut' }}
+              className="w-full h-full flex justify-center items-center"
+            >
+              <SpeakerCard
+                key={speakers[currentIndex].id}
+                name={speakers[currentIndex].name}
+                title={speakers[currentIndex].title}
+                imageUrl={speakers[currentIndex].imageUrl}
+                isActive={true}
+                handleHover={() => {}}
+                currentIndex={currentIndex}
+                pos={currentIndex}
+              />
+            </motion.div>
+          </AnimatePresence>
+        </div>
+
+        {/* Navigation */}
+        <div className="flex justify-center items-center gap-4 mt-6 md:mt-8">
+          <button
+            onClick={handlePrev}
+            className="h-9 w-9 sm:h-10 sm:w-10 md:h-12 md:w-12 border border-gray-600 rounded-full flex items-center justify-center text-gray-400 hover:bg-gray-800 hover:text-white transition-colors"
+          >
+            <GrFormPrevious className="text-2xl sm:text-3xl" />
+          </button>
+          <button
+            onClick={handleNext}
+            className="h-9 w-9 sm:h-10 sm:w-10 md:h-12 md:w-12 border-2 border-gray-400 rounded-full flex items-center justify-center text-white hover:bg-gray-800 transition-colors"
+          >
+            <GrFormNext className="text-2xl sm:text-3xl" />
+          </button>
+        </div>
```

Comment:
Dont remove it but disable this section on desktop devices (large screens). The controls should be visible only on phones. Also add a button to "See all speakers" which links to /speakers